<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>1071</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Работа с графом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddVertex">
        <property name="text">
         <string>Добавить вершину</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddEdge">
        <property name="text">
         <string>Добавить ребро</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Функции</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDFS">
        <property name="text">
         <string>Обход в глубину</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBFS">
        <property name="text">
         <string>Обход в ширину</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDijkstra">
        <property name="text">
         <string>Алгоритм Дейкстры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdj">
        <property name="text">
         <string>Матрица смежности</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
